admin/commands/pmm/client/install.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // InstallCommand is used by Kong for CLI flags and commands.
 type InstallCommand struct {
-	InstallPath  string `default:"/usr/local/percona/pmm2" help:"Path where PMM Client shall be installed"`
+	InstallPath  string `default:"/usr/local/percona/pmm3" help:"Path where PMM Client shall be installed"`
```

Comment:
can we use just `pmm` paths instead of `pmm3` since we are planning to have major releases more often.

Reply:
i.e. "/usr/local/percona/pmm" so it doesn't bear the version?